nodeshot/networking/links/models/link.py

```diff
@@ -16,6 +16,38 @@
 
 from .choices import METRIC_TYPES, LINK_STATUS, LINK_TYPES
 
+NETDIFF_PARSERS = getattr(settings, 'NODESHOT_NETDIFF_PARSERS', [
+    ('netdiff.OlsrParser', 'Olsr (jsoninfo)'),
+    ('netdiff.BatmanParser', 'Batman (Alfred)'),
+    ('netdiff.BmxParser', 'Bmx6 (q6m)'),
+    ('netdiff.NetJsonParser', 'Netjson (network-graph)'),
+    ('netdiff.CnmlParser', 'CNML'),
+    ])
+
+
+class Topology(BaseDate):
+    name = models.CharField(_('name'), max_length=75, unique=True)
+    slug = models.SlugField(max_length=75, db_index=True, unique=True, blank=True)
+    backend = models.CharField(_('backend'), max_length=128,
+        choices=NETDIFF_PARSERS,
+        help_text=_('select the netdiff's parser to parse the topology from a routing protocol'))
+    url = models.URLField(_('url'))
+    # TODO: find a way to move this in layers
+    if 'nodeshot.core.layers' in settings.INSTALLED_APPS:
```

Comment:
`nodeshot.networking.links` depends on `nodeshot.core.layers' so the `if` is not needed
reference: https://github.com/ninuxorg/nodeshot/blob/master/nodeshot/networking/links/models/__init__.py